deploy/helm/sumologic/values.yaml

```diff
@@ -2381,6 +2469,180 @@ otelagent:
 
 ## Configure otelcol
 otelcol:
+  securityContext:
+    ## ToDo: Verify following comment
+    ## The group ID of all processes in the statefulset containers. By default this needs to be otelcol(999).
+    fsGroup: 999
+
+  ## Add custom labels to all otelcol sts pods(logs, metrics, events)
+  podLabels: {}
+
+  ## Add custom annotations to all otelcol sts pods(logs, metrics, events)
+  podAnnotations: {}
+
+  ## Add custom labels to all otelcol svc (logs, metrics, events)
+  serviceLabels: {}
+
+  ## Add custom labels to all otelcol statefulset PVC (logs, metrics, events)
+  pvcLabels: {}
+
+  statefulset:
+    image:
+      repository: public.ecr.aws/sumologic/sumologic-otel-collector
+      tag: 0.0.18
+      pullPolicy: IfNotPresent
+
+  metrics:
+    enabled: true
+    config:
+      receivers:
+        telegraf:
+          agent_config: |
+            [agent]
+              interval = "2s"
+              flush_interval = "3s"
+
+            [[inputs.http_listener_v2]]
+              service_address = ":9888"
+              data_format = "prometheusremotewrite"
+              paths = [
+                "/prometheus.metrics",
+                "/prometheus.metrics.apiserver",
+                "/prometheus.metrics.applications.apache",
+                "/prometheus.metrics.applications.jmx",
+                "/prometheus.metrics.applications.kafka",
+                "/prometheus.metrics.applications.mysql",
+                "/prometheus.metrics.applications.nginx",
+                "/prometheus.metrics.applications.nginx-ingress",
+                "/prometheus.metrics.applications.postgresql",
+                "/prometheus.metrics.applications.redis",
+                "/prometheus.metrics.container",
+                "/prometheus.metrics.controller-manager",
+                "/prometheus.metrics.control-plane.coredns",
+                "/prometheus.metrics.control-plane.kube-etcd",
+                "/prometheus.metrics.kubelet",
+                "/prometheus.metrics.node",
+                "/prometheus.metrics.operator.rule",
+                "/prometheus.metrics.scheduler",
+                "/prometheus.metrics.state"
+              ]
+      extensions:
+        health_check: {}
+        # sumologic:
+        #   access_id: '{{ .Values.sumologic.accessId }}'
+        #   access_key: '{{ .Values.sumologic.accessKey }}'
+        #   collector_name: otc-{{ .Values.sumologic.collectorName | default .Values.sumologic.clusterName}}
+        #   api_base_url: '{{ .Values.sumologic.open_endpoint }}'
```

Comment:
Is there any purpose for having this commented block in `values.yaml`?  if it is added intentionally, it would be good to add comment explaining the purpose

Reply:
I've found why it is needed ;) Maybe it would be good to add link to documentation: https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Configuration.md#sumo-logic-extension or in [this](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/1702/files#diff-da997ab241c9344f2bb085822fe04e4e43eaf0f8034a9d15dc3942b8a5fbe046R2497) line mention that it is config for sumologic-otel-collector and add link to https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Configuration.md

Reply:
Just in case we go with otlp collector, but as we don't for now, I will clean it up :)